src/components/templates/ScreenShare/components/Player/Player.scss

```diff
@@ -0,0 +1,50 @@
+.video-player {
+  position: relative;
+  display: flex;
+  border: solid 0;
+  border-radius: 22px;
+  overflow: hidden;
+  width: 480px;
+  height: 320px;
+  margin: 10px;
+
+  .overlay {
+    position: absolute;
+    z-index: 2;
+    top: 0;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+
+    .buttons {
+      display: flex;
+      justify-content: flex-end;
+      margin: 10px;
+
+      .icon-container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: 30px;
+        height: 30px;
+        margin-left: 5px;
+        border: solid 0;
+        border-radius: 50%;
```

Comment:
Margins, paddings = $spacing
Border radiuses,
Font sizes, font weights should be imported from `constants.scss`